src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,11 @@
-export const UserInfo = () => <>Put the user here</>;
+export const UserInfo = ({ user: { name, email } }) => (
+  <>
```

Comment:
what is the purpose of this fragment?

Reply:
This fragment create neseccary markup for user.